Airports/EGAC/Ownership.txt

```diff
@@ -16,12 +16,3 @@ ALTOWNER:Mentoring TWR/APP:ACTM:ACT:ACAM:ACF:ACRM:ACR:AARM:AAR:STA:SL:SRE:SR:SW:
 BORDER:ACTWR
 DEPAPT:EGAC
 ARRAPT:EGAC
```

Comment:
```suggestion
ARRAPT:EGAC

;ACAPP sector ownership included in Airports/EGAA/Ownership.txt for correct ordering in the ESE
```

Maybe useful to add a comment here, just so when people come looking for the AC ownership they can find it?